assignment7/createClient.js

```diff
@@ -1,23 +1,23 @@
-var express = require("express");
+var express = require('express');
 var router = express.Router();
-var requirePromise = require("request-promise");
+var requirePromise = require('request-promise');
 
-router.get("/", function(req, res) {
+router.get('/', function(req, res) {
   var options = {
-    url: "https://api.createsend.com/api/v3.1/clients.json",
+    url: 'https://api.createsend.com/api/v3.1/clients.json',
```

Comment:
@abansal26 Create one separate file for the configuration i.e URL, user and password.

Reply:
changes made as suggested. Can be checked in commit below